src/loaders/ObjectLoader.js

```diff
@@ -306,6 +306,44 @@ THREE.ObjectLoader.prototype = {
 
 			};
 
+			var resolveReferences = function ( material, uuidStack ) {
+
+				if ( material.referencedMaterials !== undefined ) {
+
+					for ( var i = 0, l = material.referencedMaterials.length; i < l; i ++ ) {
+
+						var uuid = material.referencedMaterials[ i ];
+
+						if ( uuidStack.indexOf( uuid ) < 0 ) {
+
+							material.materials.push( materials[ uuid ].clone() );
```

Comment:
@mrdoob Do materials need to be cloned anymore?

As far as I can tell the ObjectLoader currently lets different geometry share the one material instance (i.e. no cloning). If that's correct then I should remove the clone() here.